app/packages/core/src/components/Sidebar/ViewSelection/index.tsx

```diff
@@ -240,6 +242,7 @@ export default function ViewSelection() {
           }}
           lastFixedOption={
             <LastOption
+              data-cy="create-new"
```

Comment:
Add prefix.

(also seems like a lot of these `data-cy`s are not even used, which is fine)